### PR TITLE
chore: auditing worshops, reviews, and quizzes intros

### PR DESCRIPTION
**Pull request by Ksound22**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -2218,14 +2215,14 @@
...
-          "Before you are quizzed on styling forms, you first need to review.",
+          "Before you are quizzed on styling forms, you first need to review what you've learned about it.",
           "Open up this page to review how to style form inputs, working with <code>appearance: none</code> and more."
         ]
       },
       "quiz-styling-forms": {
         "title": "Styling Forms Quiz",
         "intro": [
-          "Test what you've learned in this quiz of how to style forms using CSS."
+          "In this quix, you will test what you've learned on how to style forms."
```
Comment (moT01) [inline client/i18n/locales/english/intro.json:2225]:
```suggestion
          "In this quiz, you will test what you've learned on how to style forms."
```

File: client/i18n/locales/english/intro.json [hunk 2]
```diff
@@ -2283,14 +2279,14 @@
...
-          "Open up this page to review concepts like the <code>flex-direction</code>, <code>justify-content</code>, <code>align-items</code>, <code>flex-wrap</code> properties and more."
+          "Before you are quizzed on CSS Flexbox concepts, you first need to review what you've learned about it.",
+          "Open up this page to review concepts like the <code>flex-direction</code>, <code>justify-content</code>, <code>align-items</code>, <code>flex-wrap</code> properties, and more."
         ]
       },
       "quiz-css-flexbox": {
         "title": "CSS Flexbox Quiz",
         "intro": [
-          "Test what you've learned in this quiz of using flexbox in CSS."
+          "Test what you've learned on CSS flexbox with this quiz of using."
```
Comment (moT01) [inline client/i18n/locales/english/intro.json:2289]:
```suggestion
          "Test what you've learned on CSS flexbox with this quiz."
```

File: client/i18n/locales/english/intro.json [hunk 3]
```diff
@@ -3584,12 +3579,12 @@
...
         "title": "Bash Scripting Review",
         "intro": [
-          "Review the Bash Scripting concepts to prepare for the upcoming quiz."
+          "Review the bash scripting concepts you've learned to prepare for the upcoming quiz."
         ]
       },
       "quiz-bash-scripting": {
         "title": "Bash Scripting Quiz",
-        "intro": ["Test what you've learned in this quiz on Bash Scripting."]
+        "intro": ["Test what you've learned on bash scripting in this quiz on."]
```
Comment (moT01) [inline client/i18n/locales/english/intro.json:3587]:
```suggestion
        "intro": ["Test what you've learned on bash scripting in this quiz."]
```

**Review by moT01 — CHANGES_REQUESTED** [review]

**Review by zairahira — COMMENTED** [review]
For consistency, wherever we are referring to plurals, `"about it"` should be replaced with `"about them."` 

For example:  
- **Correct:** `"Before you are quizzed on CSS Flexbox concepts, you first need to review what you've learned about it."`  
   *Reason:* The subject, "CSS Flexbox concepts," is treated as a singular idea here.  

- **Needs Correction:** `"Before you are quizzed on CSS and accessibility, you first need to review what you've learned on it."`  
   *Suggested Change:* `"Before you are quizzed on CSS and accessibility, you first need to review what you've learned about them."`  
   *Reason:* "CSS and accessibility" refers to two distinct items, so "about them" is grammatically accurate.

**Review by zairahira — APPROVED** [review]

File: client/i18n/locales/english/intro.json [hunk 4]
```diff
@@ -2283,14 +2279,14 @@
...
+          "Before you are quizzed on CSS Flexbox concepts, you first need to review what you've learned about it.",
+          "Open up this page to review concepts like the <code>flex-direction</code>, <code>justify-content</code>, <code>align-items</code>, <code>flex-wrap</code> properties, and more."
         ]
       },
       "quiz-css-flexbox": {
         "title": "CSS Flexbox Quiz",
         "intro": [
-          "Test what you've learned in this quiz of using flexbox in CSS."
+          "Test what you've learned on CSS flexbox with this quiz."
         ]
```
[the remark this answers is not in the document]
Comment (Ksound22) [reply client/i18n/locales/english/intro.json:2290]: I can't understand why it's causing it, I have more like this.

What do I do to fix it?

[the remark this answers is not in the document]
Comment (ilenia-magoni) [reply client/i18n/locales/english/intro.json:2290]: make it all in one line like in the code in Kris's post

File: client/i18n/locales/english/intro.json [hunk 5]
```diff
@@ -3150,13 +3145,15 @@
...
-          "Before you are quizzed on working with dates, you first need to review the concepts.",
+          "Before you are quizzed on working with dates, you first need to review the concepts you've learned on it.",
           "Open up this page to review the <code>Date()</code> object and common methods."
         ]
       },
       "quiz-javascript-dates": {
         "title": "JavaScript Dates Quiz",
-        "intro": ["Test what you've learned in this quiz on JavaScript Dates."]
+        "intro": [
+          "Test what you've learned on JavaScript Dates with this quiz."
```
Comment (scissorsneedfoodtoo) [inline client/i18n/locales/english/intro.json:3153]:
```suggestion
          "Test what you've learned about JavaScript Dates with this quiz."
```

File: client/i18n/locales/english/intro.json [hunk 6]
```diff
@@ -3180,14 +3177,14 @@
...
-          "Before you are quizzed on working with audio and video in JavaScript, you first need to review.",
+          "Before you are quizzed on working with audio and video in JavaScript, you first need to review what you've learned about them.",
           "Open up this page to review concepts including the <code>Audio</code> constructor, the <code>HTMLMediaElement</code> API and more."
         ]
       },
       "quiz-javascript-audio-and-video": {
         "title": "JavaScript Audio and Video Quiz",
         "intro": [
-          "Test what you've learned in this quiz on JavaScript Audio and Video."
+          "Test what you've learned on JavaScript Audio and Video learned with this quiz."
```
Comment (scissorsneedfoodtoo) [inline client/i18n/locales/english/intro.json:3185]:
```suggestion
          "Test what you've learned about JavaScript audio and video with this quiz."
```

File: client/i18n/locales/english/intro.json [hunk 7]
```diff
@@ -3199,7 +3196,7 @@
       "workshop-plant-nursery-catalog": {
         "title": "Build a Plant Nursery Catalog",
         "intro": [
-          "In this workshop, you will practice maps and sets by building a plant nursery catalog in JavaScript."
+          "In this workshop, you will practice maps and sets by building a plant nursery catalog."
```
Comment (scissorsneedfoodtoo) [inline client/i18n/locales/english/intro.json:3197]:
```suggestion
          "In this workshop, you will practice using Maps and Sets by building a plant nursery catalog."
```

File: client/i18n/locales/english/intro.json [hunk 8]
```diff
@@ -3212,14 +3209,14 @@
...
-          "Before you are quizzed on JS Maps and Sets, you first need to review.",
+          "Before you are quizzed on JavaScript Maps and Sets, you first need to review them.",
           "Open up this page to review concepts such as the <code>Map</code> and <code>Set</code> objects, as well as WeakSet and WeakMap."
         ]
       },
       "quiz-javascript-maps-and-sets": {
         "title": "JavaScript Maps and Sets Quiz",
         "intro": [
-          "Test what you've learned in this quiz on JavaScript Maps and Sets."
+          "Test what you've learned on JavaScript Maps and Sets with this quiz."
```
Comment (scissorsneedfoodtoo) [inline client/i18n/locales/english/intro.json:3217]:
```suggestion
          "Test what you've learned about JavaScript Maps and Sets with this quiz."
```

File: client/i18n/locales/english/intro.json [hunk 9]
```diff
@@ -3252,7 +3249,7 @@
       "quiz-local-storage-and-crud": {
         "title": "Local Storage and CRUD Quiz",
         "intro": [
-          "Test what you've learned in this quiz on Local Storage and CRUD."
+          "Test what you've learned on local storage and CRUD with this quiz."
```
Comment (scissorsneedfoodtoo) [inline client/i18n/locales/english/intro.json:3250]:
```suggestion
          "Test what you've learned about local storage and CRUD with this quiz."
```

File: client/i18n/locales/english/intro.json [hunk 10]
```diff
@@ -3284,14 +3281,14 @@
       "review-javascript-classes": {
         "title": "JavaScript Classes Review",
         "intro": [
-          "Before you are quizzed on working with classes, you first need to review.",
+          "Before you are quizzed on how to work with classes, you first need to review what you've learned about them.",
```
Comment (scissorsneedfoodtoo) [inline client/i18n/locales/english/intro.json:3282]:
```suggestion
          "Before you're quizzed on how to work with classes, you should review what you've learned about them.",
```

File: client/i18n/locales/english/intro.json [hunk 11]
```diff
@@ -3284,14 +3281,14 @@
...
           "Open up this page to review concepts including the <code>this</code> keyword, class inheritance and more."
         ]
       },
       "quiz-javascript-classes": {
         "title": "JavaScript Classes Quiz",
         "intro": [
-          "Test what you've learned in this quiz on JavaScript Classes."
+          "Test what you've learned on JavaScript Classes with this quiz."
```
Comment (scissorsneedfoodtoo) [inline client/i18n/locales/english/intro.json:3289]:
```suggestion
          "Test what you've learned about JavaScript Classes with this quiz."
```

File: client/i18n/locales/english/intro.json [hunk 12]
```diff
@@ -3316,13 +3313,13 @@
...
         "intro": [
-          "Before you are quizzed on the recursion, you first need to review the concepts.",
+          "Before you are quizzed on the recursion, you first need to review the concepts you've learned about it.",
           "Open up this page to review what is recursion and what is it used for."
         ]
       },
       "quiz-recursion": {
         "title": "Recursion Quiz",
-        "intro": ["Test what you've learned in this quiz on Recursion."]
+        "intro": ["Test what your knowledge on Recursion with this quiz."]
```
Comment (scissorsneedfoodtoo) [inline client/i18n/locales/english/intro.json:3320]:
```suggestion
        "intro": ["Test your knowledge of Recursion with this quiz."]
```

File: client/i18n/locales/english/intro.json [hunk 13]
```diff
@@ -3346,14 +3343,14 @@
...
-          "Before you are quizzed on functional programming concepts, you first need to review.",
+          "Before you are quizzed on functional programming concepts, you first need to review them.",
           "Open up this page to review concepts on functional programming, currying and more."
         ]
       },
       "quiz-javascript-functional-programming": {
         "title": "JavaScript Functional Programming Quiz",
         "intro": [
-          "Test what you've learned in this quiz on JavaScript Functional Programming."
+          "Test what you've learned on JavaScript functional programming with this quiz."
```
Comment (scissorsneedfoodtoo) [inline client/i18n/locales/english/intro.json:3351]:
```suggestion
          "Test what you've learned about JavaScript functional programming with this quiz."
```

File: client/i18n/locales/english/intro.json [hunk 14]
```diff
@@ -3378,19 +3375,19 @@
...
         "intro": [
-          "Review Asynchronous JavaScript concepts to prepare for the upcoming quiz."
+          "Review asynchronous JavaScript concepts to prepare for the upcoming quiz."
         ]
       },
       "quiz-asynchronous-javascript": {
         "title": "Asynchronous JavaScript Quiz",
         "intro": [
-          "Test what you've learned in this quiz on Asynchronous JavaScript."
+          "Test what you've learned on asynchronous JavaScript with this quiz."
```
Comment (scissorsneedfoodtoo) [inline client/i18n/locales/english/intro.json:3382]:
```suggestion
          "Test what you've learned about asynchronous JavaScript with this quiz."
```

File: client/i18n/locales/english/intro.json [hunk 15]
```diff
@@ -3439,7 +3436,7 @@
       },
       "quiz-react-basics": {
         "title": "React Basics Quiz",
-        "intro": ["Test what you've learned in this quiz on React Basics."]
+        "intro": ["Test your knowledge on React basics with this quiz."]
```
Comment (scissorsneedfoodtoo) [inline client/i18n/locales/english/intro.json:3437]:
```suggestion
        "intro": ["Test your knowledge of React basics with this quiz."]
```

File: client/i18n/locales/english/intro.json [hunk 16]
```diff
@@ -3451,7 +3448,7 @@
       "quiz-react-state-and-hooks": {
         "title": "React State and Hooks Quiz",
         "intro": [
-          "Test what you've learned in this quiz on React State and Hooks."
+          "Test what you've learned on React State and Hooks with this quiz."
```
Comment (scissorsneedfoodtoo) [inline client/i18n/locales/english/intro.json:3449]:
```suggestion
          "Test what you've learned about React State and Hooks with this quiz."
```

File: client/i18n/locales/english/intro.json [hunk 17]
```diff
@@ -3470,7 +3467,7 @@
       "zagz": { "title": "282", "intro": [] },
       "quiz-advanced-react": {
         "title": "Advanced React Quiz",
-        "intro": ["Test what you've learned in this quiz on Advanced React."]
+        "intro": ["Test what you've learned on Advanced React with this quiz."]
```
Comment (scissorsneedfoodtoo) [inline client/i18n/locales/english/intro.json:3468]:
```suggestion
        "intro": ["Test what you've learned about Advanced React with this quiz."]
```

File: client/i18n/locales/english/intro.json [hunk 18]
```diff
@@ -3481,7 +3478,7 @@
       },
       "quiz-web-performance": {
         "title": "Web Performance Quiz",
-        "intro": ["Test what you've learned in this quiz on Web Performance."]
+        "intro": ["Test what you've learned on Web Performance with this quiz."]
```
Comment (scissorsneedfoodtoo) [inline client/i18n/locales/english/intro.json:3479]:
```suggestion
        "intro": ["Test what you've learned about Web Performance with this quiz."]
```

File: client/i18n/locales/english/intro.json [hunk 19]
```diff
@@ -3493,19 +3490,17 @@
       "quiz-css-libraries-and-frameworks": {
         "title": "CSS Libraries and Frameworks Quiz",
         "intro": [
-          "Test what you've learned in this quiz on CSS Libraries and Frameworks."
+          "Test what you've learned on CSS Libraries and Frameworks with this quiz."
```
Comment (scissorsneedfoodtoo) [inline client/i18n/locales/english/intro.json:3491]:
```suggestion
          "Test what you've learned about CSS Libraries and Frameworks with this quiz."
```

**Review by scissorsneedfoodtoo — COMMENTED** [review]
Hey @Ksound22, thanks for going through this file and updating all of these strings.

Sorry about the wave of suggestions here, but I tried to simplify most of the intro sentences so we don't need to worry about singular or plural wording. I also tried to normalize the quiz intros as well.

Most of these aren't required, so feel free to disregard anything, or to update the wording.

Also, if you accept some suggestions, you may need to go into the `intro.json` file and run Prettier (or save the file if you've got Prettier as an extension) so that the linter won't complain about the spacing.

**Comment by Ksound22** [comment]
> Hey @Ksound22, thanks for going through this file and updating all of these strings.
> 
> Sorry about the wave of suggestions here, but I tried to simplify most of the intro sentences so we don't need to worry about singular or plural wording. I also tried to normalize the quiz intros as well.
> 
> Most of these aren't required, so feel free to disregard anything, or to update the wording.
> 
> Also, if you accept some suggestions, you may need to go into the `intro.json` file and run Prettier (or save the file if you've got Prettier as an extension) so that the linter won't complain about the spacing.

They're all appreciated, Kris. Thank you.

File: client/i18n/locales/english/intro.json [hunk 20]
```diff
@@ -3316,13 +3311,13 @@
...
         "intro": [
-          "Before you are quizzed on the recursion, you first need to review the concepts.",
+          "Before you're quizzed on recursion, you should review what you've learned.",
           "Open up this page to review what is recursion and what is it used for."
         ]
       },
       "quiz-recursion": {
         "title": "Recursion Quiz",
-        "intro": ["Test what you've learned in this quiz on Recursion."]
+        "intro": ["Test your knowledge of Recursion with this quiz."]
```
Comment (naomi-lgbt) [inline client/i18n/locales/english/intro.json:3320]:
```suggestion
        "intro": ["Test your knowledge of recursion with this quiz."]
```

File: client/i18n/locales/english/intro.json [hunk 21]
```diff
@@ -3470,7 +3465,7 @@
       "zagz": { "title": "282", "intro": [] },
       "quiz-advanced-react": {
         "title": "Advanced React Quiz",
-        "intro": ["Test what you've learned in this quiz on Advanced React."]
+        "intro": ["Test what you've learned about Advanced React with this quiz."]
```
Comment (naomi-lgbt) [inline client/i18n/locales/english/intro.json:3468]:
```suggestion
        "intro": ["Test what you've learned about advanced React with this quiz."]
```

File: client/i18n/locales/english/intro.json [hunk 22]
```diff
@@ -3481,7 +3476,7 @@
       },
       "quiz-web-performance": {
         "title": "Web Performance Quiz",
-        "intro": ["Test what you've learned in this quiz on Web Performance."]
+        "intro": ["Test what you've learned about Web Performance with this quiz."]
```
Comment (naomi-lgbt) [inline client/i18n/locales/english/intro.json:3479]:
```suggestion
        "intro": ["Test what you've learned about web performance with this quiz."]
```

File: client/i18n/locales/english/intro.json [hunk 23]
```diff
@@ -3493,19 +3488,17 @@
       "quiz-css-libraries-and-frameworks": {
         "title": "CSS Libraries and Frameworks Quiz",
         "intro": [
-          "Test what you've learned in this quiz on CSS Libraries and Frameworks."
+          "Test what you've learned about CSS Libraries and Frameworks with this quiz."
```
Comment (naomi-lgbt) [inline client/i18n/locales/english/intro.json:3491]:
```suggestion
          "Test what you've learned about CSS libraries and frameworks with this quiz."
```

File: client/i18n/locales/english/intro.json [hunk 24]
```diff
@@ -3515,12 +3508,12 @@
...
         "title": "Typescript Review",
         "intro": [
-          "Review the Typescript concepts to prepare for the upcoming quiz."
+          "Review Typescript concepts to prepare for the upcoming quiz."
         ]
       },
       "quiz-typescript": {
         "title": "TypeScript Quiz",
-        "intro": ["Test what you've learned in this quiz on TypeScript."]
+        "intro": ["Test what you've learned on Typescript with this quiz."]
```
Comment (naomi-lgbt) [inline client/i18n/locales/english/intro.json:3516]:
```suggestion
        "intro": ["Test what you've learned on TypeScript with this quiz."]
```

It *is* capitalised like this.

File: client/i18n/locales/english/intro.json [hunk 25]
```diff
@@ -3545,7 +3538,7 @@
       },
       "quiz-bash-commands": {
         "title": "Bash Commands Quiz",
-        "intro": ["Test what you've learned in this quiz on Bash Commands."]
+        "intro": ["Test what you've learned bash commands with this quiz."]
```
Comment (naomi-lgbt) [inline client/i18n/locales/english/intro.json:3541]:
```suggestion
        "intro": ["Test what you've learned on bash commands with this quiz."]
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Review by moT01 — APPROVED** [review]

**Pull request merged**